docs/UserGuide.md

```diff
@@ -509,22 +614,27 @@ What you should see:<br>
 
 </div>
 
-Examples:
-* `find m/CS2103T CS2109S`: Returns all persons with modules CS2103T or CS2109S
-* `find n/Edward Richards`: Returns all persons with names Edward or Richards
-* `find p/9093`: Returns all persons with phone numbers starting with 9093
 
 #### Arrange contacts based on criteria `sort`
 
-With EduMate's `sort` command, you have the power to organize your contacts in a way that's most helpful for you. Whether you want to find out who shares the most modules with you or who's likely to become a close friend this semester, the `sort` command can help you out. By simply typing `sort` followed by the relevant criteria, you can sort your contacts by name, the number of shared modules, or any other specified field. This feature can help you identify potential study or social groups within your contacts, making it easier to connect and collaborate with others.
+With EduMate's `sort` command, you have the power to organize your contacts in a way that's most helpful for you. Whether you want to find out who shares the most modules with you or who's likely to become a close friend this semester, the `sort` command can help you out. 
+
+By simply typing `sort` followed by the relevant criteria, you can sort your contacts by name, the number of shared modules, or any other specified field. This feature can help you identify potential study or social groups within your contacts, making it easier to connect and collaborate with others.
 
-Formats (See [Interpreting Command Formats](#how-to-interpret-the-command-format)):
+
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Formats** (See [Interpreting Command Formats](#how-to-interpret-the-command-format)):<br>
 * `sort`: Sorts the contacts by their indices.
 * `sort z/a`: Sorts the contacts by their `z` attribute **in ascending order**
 * `sort z/d`: Sorts the contacts by their `z` attribute **in descending order**
 * `sort z/`: Sorts the contacts by their `z` attribute based on our **default ordering**
 * `sort z1/ z2/`: Sorts the contacts by their `z1` attribute, and breaks ties using their `z2` attribute
-* `sort [z/]…​`: Sorts the contacts by multiple attributes
+* `sort [z/]...​`: Sorts the contacts by multiple attributes
```

Comment:
Are we using these ellipses for everywhere? `...`? Because most of the rest should be using ZWSP as a special character.